pyscf/md/test/test_NVTBerendson.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# Copyright 2014-2022 The PySCF Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Aniruddha Seal <aniruddhaseal2011@gmail.com>
+
+import unittest
+import numpy as np
+from pyscf import gto, scf
+import pyscf.md as md
+
+CHECK_STABILITY = False
+
+def setUpModule():
+    global h2o, hf_scanner, casscf_scanner
```

Comment:
I would guess that a CASSCF is not necessary since static correlation is not strong on water.

Reply:
probably copied from the [test_velocverlet.py](https://github.com/pyscf/pyscf/blob/a27ddc77bcad42b0ee8707d93acc691fe9d51e5d/pyscf/md/test/test_velocverlet.py#L25-L55) file where I was doing some benchmarks between OpenMolcas and PySCF. Not needed in this test since it is not needed.

Reply:
Corrected!